libs/components/phone-field/src/lib/modules/phone-field/fixtures/phone-field-reactive.component.fixture.ts

```diff
@@ -39,22 +39,26 @@ export class PhoneFieldReactiveTestComponent implements OnInit {
     read: SkyPhoneFieldInputDirective,
     static: false,
   })
-  public inputDirective: SkyPhoneFieldInputDirective;
+  public inputDirective: SkyPhoneFieldInputDirective | undefined;
 
   @ViewChild(SkyPhoneFieldComponent, {
     read: SkyPhoneFieldComponent,
     static: false,
   })
-  public phoneFieldComponent: SkyPhoneFieldComponent;
+  public phoneFieldComponent: SkyPhoneFieldComponent | undefined;
 
-  public ngOnInit(): void {
-    this.phoneControl = new UntypedFormControl(this.initialValue);
+  constructor() {
+    this.phoneControl = new UntypedFormControl();
```

Comment:
[one of the unit tests needs to be able to set the initial value](https://github.com/blackbaud/skyux/blob/1abfff1d1a0f6d2c2aee12e802c874043a80d4cc/libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.spec.ts#L1340-L1352), and this cannot be done in the constructor (because the value of initialValue is undefined on construction), it must be done in the ngOnInit. so, I removed it here